app/views/posts/index.html.erb

```diff
@@ -25,7 +25,8 @@
 
   <div class="comments-container">
     <h2>Recent Comments</h2>
-    <% post.comments.each do |comment| %>
+    <% recent_comments = post.recent_comments(5) %>
```

Comment:
Let me give you a hint: 
This is not the method you have in your post model, kindly double-check and fix it.
Kindly use the method created in your post model to display just the 5 recent comments.

You're close 💪